helm_deploy/court-case-matcher/templates/requirements.yaml

```diff
@@ -0,0 +1,6 @@
+metadata:
```

Comment:
This is the only way I could get the linter to pass

Reply:
It's a warning on my machine but fails in the pipeline, presumably different versions of helm